db/migrate/20190323223752_add_first_name_and_last_name_to_users.rb

```diff
@@ -0,0 +1,10 @@
+class AddFirstNameAndLastNameToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column    :users, :first_name, :string
+    add_column    :users, :last_name,  :string
+    remove_column :users, :name,       :string
+
+    add_index     :users, :first_name
+    add_index     :users, :last_name
```

Comment:
For future reference, we could have also done `add_index :users, [:first_name, :last_name]`

Reply:
That's not quite what I wanted. They don't give the same thing.

Reply:
Oops my bad